### DISCUSSION TITLE
Property "$scopedSlots" was accessed during render but is not defined on instance.

### DISCUSSION THREAD

**Discussion by CharlesOnAir** [nuxt/nuxt#16800] [Questions]
### Environment

"nuxt": "3.0.0-rc.9",

### Reproduction

I have no idea what caused the problem, I can't provide a reproduction

### Describe the bug

I have this error : https://image.noelshack.com/fichiers/2022/36/2/1662489756-capture-d-ecran-2022-09-06-a-20-42-17.png

**Comment by sevillaarvin** [comment]
Hard to guess what's wrong here without reproduction repo, but see if this helps: https://github.com/gs-shop/vue-slick-carousel/issues/151#issuecomment-1196157228
